### ISSUE TITLE
Fresh install of rpush fails on 3.0.1 due to migration error

### ISSUE THREAD

**Issue by aj-gregory** [rpush/rpush#393]
When installing for the first time this issue arose:

```
PG::UndefinedTable: ERROR:  relation "rapns_notifications" does not exist
: ALTER TABLE "rapns_notifications" ALTER "alert_is_json" SET NOT NULL
/Users/arinhouck/Documents/beanstalk/tac-server/db/migrate/20171214163158_rpush_3_0_1_updates.rb:5:in `up'
ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "rapns_notifications" does not exist
: ALTER TABLE "rapns_notifications" ALTER "alert_is_json" SET NOT NULL
/Users/arinhouck/Documents/beanstalk/tac-server/db/migrate/20171214163158_rpush_3_0_1_updates.rb:5:in `up'
PG::UndefinedTable: ERROR:  relation "rapns_notifications" does not exist
/Users/arinhouck/Documents/beanstalk/tac-server/db/migrate/20171214163158_rpush_3_0_1_updates.rb:5:in `up'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```

Pinning the version to 3.0.0 resolved it for me

**Comment by aried3r** [comment]
I could reproduce this. Wonder why the tests didn't catch this.

Thanks for the report!

**Comment by namxam** [comment]
I am having the same issue. Any hint on how to resolve this?

**Comment by mcelicalderon** [comment]
If you want to use v3.0.1 you can just update `_rpush_3_0_1_updates` migration line 5 and 11 from `rapns_notifications` to `rpush_notifications`. `_add_rpush` migration renames the table

**Comment by aried3r** [comment]
Should be fixed in the 3.0.2 release. Feel free to reopen if it's not.
